magicctapipe/resources/config.yaml

```diff
@@ -280,8 +280,8 @@ dl2_to_dl3:
     
 cloud_correction:
     specify_cloud: yes
```

Comment:
it is not anymore "specify_cloud" option and it is not used anywhere in the code. I think you can drop it for the moment